styles/_concept.scss

```diff
@@ -25,8 +25,8 @@
 
 .topic-card__concept-article {
 	@include oTypographySans($scale: 0);
-	@include oTypographyPadding($top: 0, $bottom: 1);
-	@include oTypographyMargin($top: 2, $bottom: 0);
+	margin-top: 6px;
```

Comment:
![screenshot 2018-02-22 12 50 43](https://user-images.githubusercontent.com/295469/36539533-1be3d23a-17cf-11e8-8d76-62777cf5edf6.png)

Reply:
i know. but it feels too fat at `$top: 2` and too toit at `$top: 1`. what's a girl to do